### PR TITLE
Allow custom HTML in marker popups

### PR DESCRIPTION
**Pull request by OliverEvans96**
Hi there,

Thanks for the great plugin! I wanted to insert my own HTML into the marker popups - in my case because I want to display a table for each marker. I thought it would be worth submitting a PR in case others would like this feature, too.

Cheers!
Oliver

### CODE REVIEW

File: lib/jekyll-maps/.#google_map_tag.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+oliver@oliver-acer.549:1549418029
```
Comment (ayastreb) [inline lib/jekyll-maps/.#google_map_tag.rb]: What is this file for?

**Comment by ayastreb** [comment]
Hi,
thanks for the PR! Looks like build has failed (https://travis-ci.org/ayastreb/jekyll-maps/jobs/489898280) because of code style. Would you mind fixing that?

File: lib/jekyll-maps/.#google_map_tag.rb (hunk 1 above)
Comment (OliverEvans96) [reply lib/jekyll-maps/.#google_map_tag.rb]: Whoops! That's an emacs artifact

**Comment by OliverEvans96** [comment]
Thanks for the reply! Would you like me to squash the commits, or can you easily do that on your end?

**Pull request merged**